scalableminds/webknossos
# Show failing bucket requests

This PR fixes that errors that are thrown in a worker are not correctly transported to the main thread. This includes an update of the comlink library. This fix is used to show good error messages when bucket requests fail.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Suggestion: Test this locally:
in `webknossos/datastore/controllers/BinaryDataController.scala` add `_ <- Fox.failure("nope")` as a single line after line 62.
- Then view a dataset.
- An error toast with code 400 should appear for each layer. (The toast is not sticky, as this error could only be a temporary thing).

### Issues:
- fixes  #4725

------
